scripts/run_synth_ipbb.py

```diff
@@ -136,10 +136,10 @@ def main():
     # Setup console logging
     logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
 
-    # Check for VIVADO_BASE_DIR
-    vivado_base_dir = os.getenv('VIVADO_BASE_DIR')
+    # Check for UGT_VIVADO_BASE_DIR
+    vivado_base_dir = os.getenv('UGT_VIVADO_BASE_DIR')
     if not vivado_base_dir:
-        raise RuntimeError("Environment variable 'VIVADO_BASE_DIR' not set. Set with: 'export VIVADO_BASE_DIR=...'")
+        raise RuntimeError("Environment variable 'UGT_VIVADO_BASE_DIR' not set. Set with: 'export UGT_VIVADO_BASE_DIR=...'")
 
     # Setup console logging
     logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.DEBUG)
```

Comment:
Duplicate `logging.basicConfig`, `logging.INFO` might be fine (no `logging.debug` in this script).